### PR TITLE
Add time info to profile

### LINKED ISSUE #196: Time command improvements
> 💡 This issue is perfect for a Miunie beginner.

## Summary

Our time command would benefit from some improvements. 😊 
Feel free to implement any one (or multiple) of these.

To learn more about what the command does, read #192.

## Possible improvements
- [x] 🟢 **[Easy]** Show user's time in the Profile command **(WIP)**
- [x] 🟢 **[Easy]** Setting user's time as an Admin
- [x] 🟠 **[Medium]** Asking about user's time in relation to yours `@Miunie time 3:00 AM for @Drax`  **(WIP)**
- [x] 🟠 **[Medium]**  Asking about message post time `@Miunie time of <message-id>`
- [x] 🟠 **[Medium]**  Asking about user's time in relation to an offset `@Miunie time for @Drax in 10 hours`

### PR DESCRIPTION
**Pull request by Hellman6669**
## Related Issue

Fixes #196 

## Expected Feedback
please tix this ;-;
Drax

## Co-Authors
I send my regards to muny my, love my sky, my eternity
images
Love Hellman ;*

### CODE REVIEW

File: src/Miunie.Discord/Embeds/EmbedConstructor.cs [hunk 1]
```diff
@@ -10,7 +11,7 @@ internal static class EmbedConstructor
         public static Embed ToEmbed(this MiunieUser mUser, ILanguageProvider lang)
         {
             var realnessPhrase = lang.GetPhrase((mUser.IsBot ? PhraseKey.USER_EMBED_IS_BOT : PhraseKey.USER_EMBED_IS_HUMAN).ToString());
-
+            //<3
```
[the remark this answers is not in the document]
Comment (Hellman6669) [reply src/Miunie.Discord/Embeds/EmbedConstructor.cs]: </3

File: src/Miunie.Discord/Embeds/EmbedConstructor.cs [hunk 2]
```diff
@@ -21,9 +21,9 @@ public static Embed ToEmbed(this MiunieUser mUser, ILanguageProvider lang)
                 .AddField(lang.GetPhrase(PhraseKey.USER_EMBED_ROLES_TITLE.ToString()), string.Join("\n", mUser.Roles.Select(r => r.Name)), true)
                 .AddField(lang.GetPhrase(PhraseKey.USER_EMBED_JOINED_AT_TITLE.ToString()), $"{mUser.JoinedAt:d} at {mUser.JoinedAt:t} UTC")
                 .AddField(lang.GetPhrase(PhraseKey.USER_EMBED_CREATED_AT_TITLE.ToString()), $"{mUser.CreatedAt:d} at {mUser.CreatedAt:t} UTC", true)
+                .AddField(lang.GetPhrase(PhraseKey.USER_EMBED_TIME_TITLE.ToString()), mUser.UtcTimeOffset.HasValue ? lang.GetPhrase(PhraseKey.USER_EMBED_TIME.ToString(), DateTime.UtcNow + mUser.UtcTimeOffset) : lang.GetPhrase(PhraseKey.USER_EMBED_TIME_NOSET.ToString()), true)
                 .Build();
         }
```
Comment (petrspelos) [inline src/Miunie.Discord/Embeds/EmbedConstructor.cs]:
```suggestion
        }

```

File: src/Miunie.Discord/Embeds/EmbedConstructor.cs [hunk 3]
```diff
@@ -10,7 +11,6 @@ internal static class EmbedConstructor
         public static Embed ToEmbed(this MiunieUser mUser, ILanguageProvider lang)
         {
             var realnessPhrase = lang.GetPhrase((mUser.IsBot ? PhraseKey.USER_EMBED_IS_BOT : PhraseKey.USER_EMBED_IS_HUMAN).ToString());
```
Comment (petrspelos) [inline src/Miunie.Discord/Embeds/EmbedConstructor.cs]:
```suggestion
            var realnessPhrase = lang.GetPhrase((mUser.IsBot ? PhraseKey.USER_EMBED_IS_BOT : PhraseKey.USER_EMBED_IS_HUMAN).ToString());

```

**Comment by petrspelos** [comment]
Thank you, @Hellman6669! 😊 

bitmoji

**Pull request merged**